functions-methods.js

```diff
@@ -33,4 +49,17 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity (emailadres) {
+    if (emailadres.includes("@")) {
+        if (((emailadres.lastIndexOf(".")) === emailadres.length - 1) || (emailadres.includes(","))){
+            return false
+        } return true
+    } return false
```

Comment:
Je kan dit versimpelen:
```suggestion
    return emailadres.includes("@") && emailadres.lastIndexOf(".") !== emailadres.length - 1 && !emailadres.includes(",");
```